test/e2e/create_test.go

```diff
@@ -347,9 +343,7 @@ func (tc *testContext) disableClusterMachineApprover() error {
 	// Scale the Cluster Machine Approver Deployment to 0
 	// This is required for testing BYOH CSR approval feature so that BYOH instances
 	// CSR's are not approved by Cluster Machine Approver
-	expectedPodCount := int32(0)
-	return tc.scaleDeployment(machineApproverNamespace, machineApproverDeployment, machineApproverPodSelector,
-		&expectedPodCount)
+	return tc.scaleMachineApprover(0)
```

Comment:
not sure this refactor belongs in a bug fix but I guess it's an invasive fix anyway